ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -299,13 +316,13 @@ public OptionalLong getNextNonceForSender(final Address sender) {
     private static final AtomicLong TRANSACTIONS_ADDED = new AtomicLong();
     private final Transaction transaction;
     private final boolean receivedFromLocalSource;
-    private final Instant addedToPoolAt;
+    private final long addedToPoolAt;
     private final long sequence; // Allows prioritization based on order transactions are added
 
     TransactionInfo(
         final Transaction transaction,
         final boolean receivedFromLocalSource,
-        final Instant addedToPoolAt) {
+        final long addedToPoolAt) {
```

Comment:
We should keep this as an `Instant` - no need to change it here and having better typing is always good.